src/components/new-test/QuestionBlock.tsx

```diff
@@ -5,15 +5,22 @@ import { SuperSelect } from './SuperSelect';
 import { SuperInput } from './SuperInput';
 import { Timer } from './Timer';
 
+
 type QuestionBlockType = {
   value: string;
+  difficultyValue: string;
   handleTypeChange: (event: SelectChangeEvent) => void;
+  handleDifficultyChange: (event: SelectChangeEvent) => void;
   items: string[];
+  difficultyItems: string[];
 };
 
 export const QuestionBlock = ({
   value,
+  difficultyValue,
   handleTypeChange,
+  difficultyItems,
```

Comment:
looks like you don't need drill difficulty Items throw all components. Just take them from the store

Reply:
use useAppSelector to resolve this task